src/Extensions/Hosting.Services/Internal/ServiceInitializationEventSource.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Omex.Extensions.Hosting.Services
 	/// <summary>
 	/// Service Fabric event source
 	/// </summary>
-	[EventSource(Name = "Microsoft-OMEX-ServiceInitializationLogs")] //Breaking Change: event source renamed
+	[EventSource(Name = "Microsoft-OMEX-ServiceInitializationLogs")] //TODO: new event source should be registred
```

Comment:
I'm not sure about the github conventions, but we usually add PBIs to all TODO comments

Reply:
Makes sense, I've create an issue and added it's number